### DISCUSSION TITLE
Does WAHA support the 8-character code device linking method from WhatsApp?

### DISCUSSION THREAD

**Discussion by MohamedESalem** [devlikeapro/waha#563] [Q&A] [answered]
I am asking if WAHA currently supports WhatsApp's  8-character code method for linking devices, as opposed to using a QR code.

**Comment by devlikepro** [comment, marked as answer]
Yeah, in both engines it's available starting 2024.9.6 version!
Use API https://waha.devlike.pro/docs/how-to/sessions/#get-pairing-code
```
POST /api/{session}/auth/request-code
{
  "phoneNumber": "12132132130"
}
```

Or Dashboard to get it
![image](https://github.com/user-attachments/assets/4e35f19b-af58-42b3-b20c-be34478e5c7f)


[![patron:PRO](https://img.shields.io/badge/patron-PRO-188a42)](https://waha.devlike.pro/docs/how-to/plus-version/#tiers)
